src/components/users.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Users } from '../mocks/users'
+
+class UsersList extends Component {
+	constructor(){
+		super();
+		this.state = {
+			data: Users
+		}
+	}
+  render() {
+    return (
+      <div>
+        <Header/>
+        <table className="table table-striped table-hover">
+        	<thead>
+				    <tr>
+				      <th>ID</th>
+				      <th>First Name</th>
+				      <th>Last Name</th>
+				    </tr>
+				  </thead>
+        	<tbody>
+        	 	{this.state.data.map((person, i) => <TableRow key = {i} data = {person} />)}
+        	</tbody>
+        </table>
+      </div>
+    );
+  }
+}
+class Header extends Component{
```

Comment:
No multiclasses in one file
remover class Header to it's own folder
each component should have own folder